.github/workflows/build.yml

```diff
@@ -0,0 +1,97 @@
+name: Build
+
+on: [push]
+
+jobs:
+  test_framework:
+    name: "Test framework (iOS ${{ matrix.version.ios }})"
+    runs-on: macos-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - {ios: 13.5, watchos: 6.2.1}
```

Comment:
It is worth expanding this to minimum supported version if possible? If not here, then the demo apps? Not sure if we'd get compile or runtime (test) errors with unsupported versions but would be good to get confidence by testing across a range.

Reply:
I agree, I'd like and I've tried... but failed.
I've put that on `matrix` for that reason but unfortunately there aren't many version supported: https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md#installed-sdks
I hope in the future they will enlarge the number of iOS versions supported.

Reply:
Ah ok, I'm sure this will increase over time. 

Could maybe do 12.4 and 13.5? That way, at least we're comfortable with running across multiple versions.
Not essential though, I'm happy to approve either way.